src/plugins/intel_cpu/src/node.cpp

```diff
@@ -907,9 +916,6 @@ MemoryPtr Node::prepareWeightMemory(DnnlMemoryDescPtr dstWeightDesc, DnnlMemoryD
 }
 
 void Node::toNumaNode(int numaNodeID) {
```

Comment:
`isExecutable()` costs a bit more than assumption.
Another relatively simple and quick fix is to check the member variable `curNumaNode` with `numaNodeID` before executing `isExecutable()` here. And then other changes are not needed.